rotkehlchen/chain/ethereum/decoding/pickle/decoder.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Optional
+
+from rotkehlchen.accounting.structures import (
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.decoding.pickle.constants import PICKLE_CONTRACTS
+from rotkehlchen.chain.ethereum.decoding.structures import ActionItem
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.constants.ethereum import ZERO_ADDRESS
+from rotkehlchen.types import EthereumTransaction
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+
+def maybe_enrich_pickle_transfers(
+    token: Optional[EthereumToken],  # pylint: disable=unused-argument
```

Comment:
```suggestion
    token: EthereumToken,  # pylint: disable=unused-argument
```

I already told you in the previous review. The token here is not optional.